src/components/Debugger/Debugger.scss

```diff
@@ -0,0 +1,53 @@
+button.pf-c-button.chr-c-button-debugger {
+  svg {
+    transform: rotate(90deg);
+  }
+  position: fixed;
+  z-index: 100;
+  padding: 5px;
```

Comment:
I would try using the PF variables as much as possible. You can find them here: https://www.patternfly.org/v4/developer-resources/global-css-variables#global-css-variables